vllm/entrypoints/openai/cli_args.py

```diff
@@ -134,6 +134,11 @@ def make_arg_parser(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
         help="When --max-logprobs is specified, represents single tokens as"
         "strings of the form 'token_id:{token_id}' so that tokens that"
         "are not JSON-encodable can be identified.")
+    parser.add_argument("--keep-alive-on-engine-death",
```

Comment:
Only minor but I wonder if this would be better as an env var since I think it would only be used in debugging scenarios...

Reply:
Ah, yeah I can change that. I only recently learned that there's an explicit split between the cli args and `vllm.envs`